frontend/src/ladder/components/InfoBox.vue

```diff
@@ -201,8 +217,163 @@ function promote(event) {
   } else {
     stompClient.send("/app/ladder/post/promote", { event: event });
   }
+}
+
+// TODO: Remove Difference in asshole-event and promote
```

Comment:
The Whole ETA Stuff doesn't need to be Part of the InfoBox, it can be its onw file, like etaUtils and being imported so it can be used at other places outside the infobox as well